startup/gui/menus.py

```diff
@@ -235,22 +235,24 @@ def __lightCreator( nodeName, shaderName, shape ) :
 		import GafferOSL
 		import GafferOSLUI
 
-		GafferAppleseedUI.ShaderMenu.appendShaders( nodeMenu.definition() )
+		if os.environ.get( "GAFFERAPPLESEED_HIDE", "" ) != "1" :
```

Comment:
Can we have something more specific, like GAFFERAPPLESEED_DISABLE_NODE_MENU?

Reply:
I had that first, but It also removes the help menu entry...

Reply:
As per off-line chat, moved to `GAFFERAPPLESEED_HIDE_UI` in https://github.com/GafferHQ/gaffer/commit/4803965593c4e9fcc456fbe3f8bd9497eb1f6f7c, and also now hide Diagnostic shading modes, output and ShaderTweak presets.